plotter/db.py

```diff
@@ -671,11 +671,8 @@ def all_profiles_ready():
 
 class Profile(Resource):
     keys = ("name", )
-    RELATED = (
-        "Regions", "Profile", "Models",
-        "AnnotationCounts", "DisplayedProfile",
-        "ChromProbes", "ModelError",
-        )
+    RELATED = ("Breakpoints","Copies", "Profile", "Models", "AnnotationCounts", "DisplayedProfile","ChromProbes", "ModelError",)
```

Comment:
two comments about why it would be better to maintain the line breaks, instead of putting this one one big line:
1. it is easier to see the difference (deleted regions and added breakpoints and copies)
2. easier to read in general (80 char width max for each line please)